roomranger/src/main/java/org/launchcode/roomranger/models/User.java

```diff
@@ -2,57 +2,25 @@
 
 import jakarta.persistence.*;
 import java.util.Set;
-//CHANGED LONG ID TO INT ID
+
 @Entity
-public class User {
+public class User extends AbstractEntity{
```

Comment:
Sonal is also using this table, I am not sure it is fine if you change this class

Reply:
Ok.. after Sonal's review I will change it